core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/dremio/DremioDatabaseConnectionService.java

```diff
@@ -27,4 +27,9 @@ public String getDriver(DremioDatabaseConnection databaseConnection) {
     public Class<DremioDatabaseConnection> appliesTo() {
         return DremioDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(DremioDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class